### ISSUE TITLE
Change of model

### ISSUE THREAD

**Issue by zczlsde** [AI4Finance-Foundation/FinRL#1088]
Hi, is it possible to change the structure of model instead of using the model provided in the library

**Comment by mmmarchetti** [comment]
Yes, one way is instead of using "pip install", clone the repository and make desired changes.
